src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -27,6 +27,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)ConvertProgram\ConvertProgramAnalysis_ProgramMain.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)ConvertProgram\ConvertToTopLevelStatementsDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)ConvertTypeofToNameof\CSharpConvertTypeOfToNameOfDiagnosticAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Copilot\CSharpImplementNotImplementedExceptionDiagnosticAnalyzer.cs" />
```

Comment:
I don't think we want this analyzer in the shared project. Since every analyzer here are also shipped in the code-style analyzer package. Given the codefix is in `Features` and this analyzer is only used in combination of the fixer, it should be moved to `Features` as well

Reply:
`CSharpAnalyzers` project where I added this is already under the Features layer.